specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2921 @@
+                {
+                  "$ref": "#/parameters/ApiVersionParameter"
+                },
+                {
+                  "description": "Collection of documents to analyze and tasks to execute.",
+                  "in": "body",
+                  "name": "body",
+                  "schema": {
+                    "$ref": "#/definitions/AnalyzeTextJobsInput"
+                  }
+                }
+              ],
+              "responses": {
+                "202": {
+                  "description": "A successful call results with an Operation-Location header used to check the status of the analysis job.",
+                  "headers": {
+                    "Operation-Location": {
+                      "type": "string"
+                    }
+                  }
+                },
+                "default": {
+                  "description": "Error response.",
+                  "schema": {
+                    "$ref": "#/definitions/ErrorResponse"
+                  }
+                }
+              },
+              "x-ms-examples": {
+                "Successful Submit Analysis Job Request": {
+                  "$ref": "./examples/SuccessfulAnalyzeTextJobSubmitRequest.json"
+                }
+              },
+              "x-ms-long-running-operation": true
+            }
+          },
+        "/analyze-text/jobs/{jobId}": {
+            "get": {
+              "produces": [
+                "application/json"
+              ],
+              "description": "Get the status of an analysis job.  A job may consist of one or more tasks.  Once all tasks are completed, the job will transition to the completed state and results will be available for each task.",
+              "operationId": "AnalyzeText_JobStatus",
+              "summary": "Get analysis status and results",
+              "parameters": [
+                {
+                  "$ref": "#/parameters/ApiVersionParameter"
+                },
+                {
+                  "$ref": "#/parameters/JobId"
+                },
+                {
+                  "$ref": "#/parameters/ShowStats"
+                },
+                {
+                  "default": 20,
+                  "description": "(Optional) Set the maximum number of results per task. When both top and skip are specified, skip is applied first.",
+                  "in": "query",
+                  "maximum": 50,
+                  "minimum": 1,
+                  "name": "top",
+                  "type": "integer",
+                  "format": "int32"
+                },
+                {
+                  "default": 0,
+                  "description": "(Optional) Set the number of elements to offset in the response. When both top and skip are specified, skip is applied first.",
+                  "in": "query",
+                  "minimum": 0,
+                  "name": "skip",
+                  "type": "integer",
+                  "format": "int32"
+                }
```

Comment:
From the response schema `AnalyzeTextJobState`, this seems not a List API. How does `top` and `skip` apply, to `tasks.entityRecognitionTasks`?

Reply:
Thanks for pointing this out.  It was a missed update of the old design where we used property names within an object instead of the discriminator pattern for a heterogeneous list.  I will update it shortly.

Reply:
I have updated this now.  Let me know what you think.